packages/hawtio/src/plugins/shared/chart/Chart.tsx

```diff
@@ -181,39 +195,60 @@ export const Chart: React.FunctionComponent = () => {
     )
   }
 
-  const AttributeChart = ({ name, data }: { name: string; data: { name: string; x: number; y: number }[] }) => (
-    <div style={{ height: '140px', display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
-      <div style={{ width: '20%', overflow: 'hidden', fontSize: 10, paddingRight: 10, flexGrow: 1, flexShrink: 0 }}>
-        {name}
-      </div>
-      <div style={{ width: '80%', maxWidth: '80%', height: '140px', flexGrow: 0, flexShrink: 0 }}>
+  const AttributeChart = ({
```

Comment:
this is now too big to have it inside a component definition. push it out to the top level of the same file. making it a independent component is also better for isolating component rendering as you know.